setup-homebrew/action.yml

```diff
@@ -4,6 +4,11 @@ author: dawidd6
 branding:
   icon: settings
   color: yellow
+inputs:
+  test-bot:
+    description: Install the `homebrew/test-bot` tap.
+    required: false
+    default: true
```

Comment:
I'd actually rather we didn't remove this. You need `brew test-bot --cleanup-before` for `brew doctor` to not fail.